packages/app/i18n/locales/en_US/translation.json

```diff
@@ -0,0 +1,7 @@
+{
+  "today": "Today at {0}",
+  "sidecarExpand": "Expand to full width",
+  "sidecarMinimize": "Minimize the sidecar",
+  "sidecarRestore": "Restore split screen",
+  "screenshotBalloon": "Capture screenshot"
+}
```

Comment:
missing Close; also the Minimize text is now just "Minimize"

Reply:
Thanks! Updated